### ISSUE TITLE
Disallow NULL in Specimen_event.collecting_source?

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#2080]
Specimen_event.collecting_source is NULLable, and the code table contains "unknown." Is there a need for "we implicitly don't know" and "we explicitly don't know" or something, or can we merge those one way or the other?

_Originally posted by @dustymc in https://github.com/ArctosDB/arctos/issues/1942#issuecomment-489680458_

I say we require an answer and if none is given, we enter unknown.

**Comment by AJLinn** [comment]
> I say we require an answer and if none is given, we enter unknown.

There is no equivalent for this for cultural collections. Please do not require us to enter information into a field that is irrelevant for our holdings. By automatically stating "unknown" it makes it seem like we're missing data when in fact, it is just data that is not translatable to cultural collections.

**Comment by campmlc** [comment]
should "N/A" be an option?

**Comment by dustymc** [comment]
Right, I need to amend my question.  Is there a need for "we have nothing to say" and "we explicitly don't know," or can we merge those one way or the other?

**Comment by Jegelewicz** [comment]
So, turn it around - remove "unknown" but allow NULL.

**Comment by dustymc** [comment]
merge-->https://github.com/ArctosDB/arctos/issues/3997
